package.json

```diff
@@ -11,6 +11,7 @@
     "@rollup/rollup-linux-x64-gnu": "4.6.1"
   },
   "dependencies": {
+    "inter": "^2.1.0",
```

Comment:
Is it possible to use: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DisplayNames? Instead of pulling a 80MB library?

```js
const displayNames = new Intl.DisplayNames(['es'], { type: 'region' });
const displayName = displayNames.of('ES');
console.log(displayName);
```

Reply:
It doesn't support the 4 letter locale codes. Intl only supports the old 2 letter locales, and only has locale names in english, making it useless for dynamic locales.